plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/TransactionLogAccess.java

```diff
@@ -98,6 +103,9 @@ public class TransactionLogAccess
     private final Cache<TableLocation, TableSnapshot> tableSnapshots;
     private final Cache<TableLocation, DeltaLakeDataFileCacheEntry> activeDataFileCache;
 
+    // TODO move to query-level state
```

Comment:
What does this TODO actually mean?
It is not clear to me what query-level state means.

Reply:
for example
TLA is a singleton
DLM is a query-level (transaction-level) thing